simulation/src/node.ts

```diff
@@ -89,6 +89,14 @@ export class Node {
     this.aggregates = {}
   }
 
+  aggregationId(child: string[]): string {
+    const s = cloneDeep(child).sort().join('-')
+    return s.split('').reduce((a, b) => {
+      a = ((a << 5) - a) + b.charCodeAt(0)
+      return a & a
+    }, 0).toString()
+  }
```

Comment:
That's kind obscure to me :sweat_smile: 
What does it do? A bit of jsdoc/comments might be useful here